src/hubbleds/components/dotplot_viewer/dotplot_viewer.py

```diff
@@ -366,15 +366,16 @@ def on_zoom(bounds_old, bounds_new):
                 
             line_marker_at.subscribe(lambda new_val: _update_lines(value = new_val))
             vertical_line_visible.subscribe(lambda new_val: _update_lines())
-            def reset_hist_n_bin():
+            def reset_hist_n_bin(xmin = None, xmax = None):
+                print("reset_hist_n_bin")
```

Comment:
I guess this should use the logger here (or it's just a debugging statement that we want to remove).

Reply:
Removed that line. It was just for debugging.